### PR TITLE
[dev-launcher][dev-menu] cleanup UI 

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

This PR includes a couple small UI changes and fixes after QA-ing on a smaller screen size (iPhone SE) 
Also includes a fix for the "Copy" button on the dev menu which was not working

# How

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export async function getDevSessionsAsync({
     devSessions = devSessions.concat(sessions);
   }
 
-  if (!devSessions.length && !isDevice) {
```
Comment (esamelson) [inline packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts:32]: in light of our conversation earlier -- was this an intentional change?

**Review by esamelson — APPROVED** [review]
👌

File: packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts (hunk 1 above)
Comment (ajsmth) [reply packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts:32]: nope totally missed this - thanks for flagging I'll revert

File: packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export async function getDevSessionsAsync({
...
+  if (!devSessions.length) {
```
Comment (ajsmth) [inline packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts:32]:
```suggestion
  if (!devSessions.length && !isDevice) {
```

**Pull request merged**